<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>681</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dictionary Tree QAQ</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QGroupBox" name="insertNewWord_GroupBox">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>225</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="tabletTracking">
           <bool>false</bool>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>addupNewWord</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <item>
              <widget class="QTextEdit" name="UserInput_textEdit">
               <property name="font">
                <font>
                 <family>Bahnschrift Light</family>
                </font>
               </property>
               <property name="locale">
                <locale language="Chinese" country="China"/>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="AddupWords_Btn">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>36</height>
                  </size>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="text">
                  <string>addupWords</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="ClearInput_Btn">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>36</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>clearText</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>16</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>225</height>
           </size>
          </property>
          <property name="title">
           <string>searchExistedWord</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="sizeConstraint">
              <enum>QLayout::SetMaximumSize</enum>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Search:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="searchWord_lineEdit">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>24</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>The results will be listed here&gt;&gt;&gt;   (by preffix)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListView" name="showWord_listView"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="output_GroupBox">
        <property name="title">
         <string>outputInfo are here&gt;&gt;&gt;   (by lexical order)</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QPushButton" name="resetTree_Btn">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string>resetTree</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="updateOutput_Btn">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string>↓update this list↓</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="Output_textEdit">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <family>Bahnschrift Light</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="tabletTracking">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size: 16px;
</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Bahnschrift Light'; font-size:16px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Words will be listed here, &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;	(First valid then invalid)&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:12pt; color:#008000;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true"/>
    </item>
    <item>
     <widget class="QLabel" name="Status_label">
      <property name="text">
       <string>StatusInfo</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>944</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuovo">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionopen_file"/>
   </widget>
   <widget class="QMenu" name="menusearch">
    <property name="title">
     <string>WordLib</string>
    </property>
    <addaction name="actionsearch_in_WordLib"/>
    <addaction name="actioncheck_userinput_in_WordLib"/>
   </widget>
   <addaction name="menuovo"/>
   <addaction name="menusearch"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionopen_file">
   <property name="text">
    <string>open file...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionsearch_in_WordLib">
   <property name="text">
    <string>search in WordLib</string>
   </property>
  </action>
  <action name="actioncheck_userinput_in_WordLib">
   <property name="text">
    <string>check userinput in WordLib</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ClearInput_Btn</sender>
   <signal>clicked()</signal>
   <receiver>UserInput_textEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetTree_Btn</sender>
   <signal>clicked()</signal>
   <receiver>ClearInput_Btn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddupWords_Btn</sender>
   <signal>clicked()</signal>
   <receiver>Status_label</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>643</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
